pkg/resolver/resolver.go

```diff
@@ -143,11 +159,25 @@ func GetLocalPath(repo string, chartpath string) (string, error) {
 		depPath = filepath.Join(chartpath, p)
 	}
 
-	if _, err = os.Stat(depPath); os.IsNotExist(err) {
-		return "", fmt.Errorf("directory %s not found", depPath)
-	} else if err != nil {
-		return "", err
+	return depPath, nil
+}
+
+//IsLocalSubChart checks if file repo path does belong to the chartpath "/charts" folder
+func IsLocalSubChart(repo string, chartpath string) (bool, error) {
+	if !strings.HasPrefix(repo, "file://") {
+		return false, nil
 	}
+	var isSubChart bool
 
-	return depPath, nil
+	destPath := filepath.Join(chartpath, "charts")
+	origPath, err := GenerateLocalPath(repo, chartpath)
+	if err != nil {
+		return false, err
+	}
+	relPath, err := filepath.Rel(destPath, origPath)
+	if err != nil {
+		return false, err
+	}
+	isSubChart = !strings.HasPrefix(relPath, ".")
```

Comment:
you can simply call `return !strings.HasPrefix(relPath, "."), nil` to save an allocation on the call stack.